src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url('../images/star.svg');
```

Comment:
The URL path for the background image should be checked to ensure it is correct and the image exists at the specified location.